{"name": "Негативная проверка авторизации курьера с неверным заполнением обязательного поля \"password\"", "status": "passed", "description": "Запрос должен вернуть ошибку и правильный код ответа", "attachments": [{"name": "stdout", "source": "dd146e43-0284-45d9-87b9-bba493e44cdd-attachment.txt", "type": "text/plain"}], "start": 1721557345657, "stop": 1721557346168, "uuid": "5ca3a7fd-71a4-41ce-b08c-f2e4d8299c8b", "historyId": "9567dfa792c64daa9aee756c5d10675d", "testCaseId": "9567dfa792c64daa9aee756c5d10675d", "fullName": "test_authorization_courier.test_authorization_with_wrong_password.TestAutorizationCourierWithWrongPassword#test_authorization_courier_with_wrong_password_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('reg')"}, {"name": "parentSuite", "value": "test_authorization_courier"}, {"name": "suite", "value": "test_authorization_with_wrong_password"}, {"name": "subSuite", "value": "TestAutorizationCourierWithWrongPassword"}, {"name": "host", "value": "DESKTOP-IKUPUFD"}, {"name": "thread", "value": "4440-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_authorization_courier.test_authorization_with_wrong_password"}]}